cognite/neat/core/_utils/rdf_.py

```diff
@@ -296,3 +297,16 @@ def uri_display_name(thing: URIRef) -> str:
     elif "http://purl.org/cognite/neat/data-model/" in thing:
         return "NEAT(" + ",".join(thing.replace("http://purl.org/cognite/neat/data-model/", "").split("/")) + ")"
     return remove_namespace_from_uri(thing)
+
+
+def uri_to_cdf_id(uri: URIRef) -> str:
+    """Convert a URI to a user-friendly string.
+    Removing the namespace and unquoting the URI.
+    If the namespace is a CDF space, then it will be included in the string as space:externalId.
+    """
+    namespace, entity_id = split_uri(uri)
+    output_str = urllib.parse.unquote(entity_id)
```

Comment:
Might be me being paranoic here.

urlib.parse.unquote might misbehave in some cases. Such as:

```
urllib.parse.unquote("neat%cool")
output: "neat%cool"
```
🔝 it will silently do nothing here, won't fail or give you an exception.

Is it possible to happen in our URIs? If it is, then we will need to use another parser here.

Reply:
I don't think this is an issue. This is essentially the reverse of quote which we run when writing to the triple. So if you run 
```python
>>> urllib.parse.unquote(urllib.parse.quote("neat%cool")
'neat%cool'
```
with the intermediate being `neat%25cool`

Reply:
👍 I was just being a bit paranoic =P